Various Bug Fixes See Description

Newt-Tea opened the pull request:
Bug Fix Notes
- Fixed URL Pathing in project and app level urls.py
- Made a temporary Logout view and HTML Template to avoid a call to a non-existant view
- Added manual Grouping and user Permissions to the user class to avoid conflict with the AbstractUser Class

It appears everything should be functional after the forms are created @aambrose1

Newt-Tea committed:
Various Bug Fixes See PR #48

mccrosno reviewed (commented):
In models.py, I believe that the added text is to fix the conflicting related_name fields that creates an error when trying to run the server. To fix, all that is needed is to add AUTH_USER_MODEL value to settings.py

mccrosno committed:
Updated settings.py

Added AUTH_USER_MODEL

mccrosno committed:
Fixed bugs found in PR #48

CHANGES

eMarket folder:
urls.py - changed second path from including my_app.urls to app.urls
settings.py - added AUTH_USER_MODEL to settings to create custom user model

app folder:
forms.py - removed User import from auth.models, changed import UserProfile to User
views.py - removed User import from auth.models, changed import UserProfile to User
product_list.html - added a placeholder file to satisfy path error that was occuring
urls.py - temporarily commented out logout path, changed product_list url path

mccrosno reviewed (approved):
Bugs fixed
